hudi-flink/pom.xml

```diff
@@ -173,5 +173,102 @@
       <artifactId>bijection-avro_${scala.binary.version}</artifactId>
       <version>0.9.7</version>
     </dependency>
+
+    <!-- Test -->
```

Comment:
`Test suite` or `Test frameworks` sounds better?

Reply:
changed to `Junit Test Suite`